src/datacenters/parser.ts

```diff
@@ -1,17 +1,21 @@
-import Ajv2019 from 'ajv/dist/2019.js';
+import Ajv2019 from 'https://esm.sh/v124/ajv@8.11.0/dist/2019.js';
```

Comment:
I assume there's some reason for these import changes but i'm curious what the reason is (and if we should write it somewhere)

Reply:
There were conflicting problems with building the node module and running Deno processes when we used the version in the import map, though I forget what the exact errors were at this point. This was the only solution I found where both processes were happy

Reply:
Got it, figured it may have been related to that, given we're unlikely to need to import this anywhere else anyways this is probably fine